app/views/docs/installation.phtml

```diff
@@ -1,3 +1,8 @@
+<div class="notice margin-bottom">
+    <h3>Migration Instructions</h3>
+    <p>If you are migrating from an older version of Appwrite, you need to follow the migration instructions <a href="/docs/upgrade" target="blank" rel="noopener">here.</a></p>
```

Comment:
```suggestion
    <p>If you are migrating from an older version of Appwrite, you need to follow the <a href="/docs/upgrade">migration instructions</a></p>
```

Better for SEO to have a meaningful text as the link. No need to for a new tab or noopener setting for a local link